src/seedu/addressbook/Main.java

```diff
@@ -24,7 +24,7 @@
 
     /** Version info of the program. */
     public static final String VERSION = "AddressBook Level 2 - Version 1.0";
-
+    private static String[] NOTE;
```

Comment:
Should probably have Notes as its own class instead of being in Main.

Reply:
Agreed. A proper design would require Note to be a class which associates itself to a Person.